transport-interop/helpers/cache.ts

```diff
@@ -62,6 +62,11 @@ switch (modeStr) {
             const cacheKey = await hashFiles(files)
             console.log("Cache key:", cacheKey)
 
+            if (AWS_BUCKET === "") {
```

Comment:
This will catch `undefined`, `null` and `''`.  As written it'll only catch `''`.

```suggestion
            if (!AWS_BUCKET) {
```